src/Microsoft.Net.Http.Headers/SetCookieHeaderValue.cs

```diff
@@ -90,42 +90,53 @@ public string Value
         public override string ToString()
         {
             StringBuilder header = new StringBuilder();
+            AddStringValue(header);
 
-            header.Append(_name);
-            header.Append("=");
-            header.Append(_value);
+            return header.ToString();
+        }
+
+        /// <summary>
+        /// Add string representation of this <see cref="SetCookieHeaderValue"/> to given <paramref name="builder"/>.
+        /// </summary>
+        /// <param name="builder">
+        /// The <see cref="StringBuilder"/> to receive the string representation of this
+        /// <see cref="SetCookieHeaderValue"/>.
+        /// </param>
+        public void AddStringValue(StringBuilder builder)
```

Comment:
This name is confusing. `ToStringBuilder`? `CopyToStringBuilder`? `SerializeToStringBuilder`? `AppendToStringBuilder`?